lpython_tests.py

```diff
@@ -41,6 +43,130 @@ def test_namedexpr():
     while a := 1:
         y = 1
 
+
+def test_factorial_1(x: i32) -> i32:
+    if x < 0:
+        return 0
+    result: i32
+    result = 1
+    while x > 0:
+        result = result * x
+        x -= 1
+    return result
+
+
+def test_factorial_2factorial(x: i32) -> i32:
+    result: i32
+    result = 1
+    i: i32
+    for i in range(0, x+1):
+        result = result * i
+    return result
+
+
+def test_pow():
+    a: i32
+    a = pow(2, 2)
+
+
+def test_pow_1(a: i32, b: i32) -> i32:
+    res: i32
+    res = pow(a, b)
+    return res
+
+
+def test_for():
+    i: i32
+    for i in range(0, 10):
+        if i == 0:
+            continue
+        if i > 5:
+            break
+
+
+def test_augassign():
+    r: i32
+    s: i32
+    r = 0
+    r += 4
+    s = 5
+    r *= s
+    r -= 2
+    s = 10
+    r /= s
+    a: str
+    a = ""
+    a += "test"
+
+
+def test_ord_chr():
+    s: str
+    a: i32
+    a = ord('5')
+    s = chr(43)
+
+
+def test_assert():
+    a: i32
+    a = 5
+    assert a == 5
+    assert a == 10
```

Comment:
```suggestion
    assert a != 10
```
`asr_verify.cpp` should check this

Reply:
Probably, that will be tested only when we run on some backend.